### DISCUSSION TITLE
Profiles: Flash or flicker to indicate actions

### DISCUSSION THREAD

**Discussion by Cryptid-9527** [runelite/runelite#15996] [Ideas]
I'd like to see a flash or flicker when a profile is swapped.

![image](https://user-images.githubusercontent.com/58305537/205750903-62b59dc9-f281-4677-938a-fedaa2ffd13b.png)

![image](https://user-images.githubusercontent.com/58305537/205751288-7cb9d22e-59c8-487f-b466-f161449d5456.png)

**Comment by Nightfirecat** [comment]
Support for hub plugins should be directed to the author of the plugin.
You can find the support link by searching for the plugin in the Plugin Hub panel and clicking the `?` button on the plugin, or by right-clicking the plugin in the plugin panel and clicking the `Support` menu option.
